docs/source/sparse.rst

```diff
@@ -363,6 +363,26 @@ assumption that the fill value is negative infinity.
 
 .. See https://github.com/Quansight-Labs/rfcs/tree/pearu/rfc-fill-value/RFC-0004-sparse-fill-value for a new API
 
+.. _sparse-csr-docs:
+
+Sparse CSR Tensor
++++++++++++++++++
+
+The CSR (Compressed Storage Row) sparse tensor implements the CSR sparse tensor
+format.
+
+Construction
+------------
+
+As a result of pytorch being linked with 32-bit MKL, we must use 32 bit
+integer indices in the ``crow_indices`` and ``col_indices`` functions.
+Therefore, a reduction scheme where the resulting dimensions exceed ``INT_MAX``
+for 32 bit integers is not supported (as of now).
```

Comment:
This is an implementation detail, expose it as a note.

Reply:
Shouldn't the user be made aware that they should pass the indices as 32-bit tensors? I can change the comment to show that the construction should happen with 32-bit for so and so reasons. I think that would let curious users look at the reason for such a indexing scheme.

Reply:
Just say that indices must be 32-bit. The reason can be explained elsewhere (in code, in an advanced section note, or similar)